### PR TITLE
React version of ViewBovis

### PR DESCRIPTION
**Pull request by praveenmurugan101**
- React app initialisation for ViewBovis
1.  SNP matrix button and tab removed in home page content and navbar.
2. no more toggle movement lines option in Cattlemovement map, updated react version of arrow head which is slightly different in appearance.
3. no more SNP matrix page, appearance changes in SNP map icons and legends, different highlight styling, clustered samples expand on opening sidebar snp table, bootstrap version of Tabulator. 
4. Basemap option expand only on mouse over, tooltip display in Risk area layers on mouse over
5.  using classic responsive table instead of Tabulator in nextstrain
6. updated links in Help and support page
- Flask changes for serving React build and manifest files

- Flask fixes for keyerror in latlon table dictionary object (sample: AF-12-01052-23, snpdistance > 1)

### CODE REVIEW

File: app/build/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<!doctype html><html lang="en"><head><meta charset="utf-8"/><link rel="icon" href="/favicon.ico?"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="theme-color" content="#000000"/><meta name="description" content="Visualisation tool for Mycobacterium bovis"/><link rel="apple-touch-icon" href="/logo192.png"/><link rel="manifest" href="/manifest.json"/><title>ViewBovis</title><script defer="defer" src="/static/js/main.4a67c324.js"></script><link href="/static/css/main.ba499754.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"></div></body></html>
```
Comment (ellisrichardj) [inline app/build/index.html:1]: Can I suggest 'content' is updated to match the 'name' in `manifest.json`: "Visualisation of Mycobacterium bovis transmission"

File: app/front_end/src/components/HelpSupport/HelpSupport.jsx [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+              </a>
+            </p>
+            <p>
+              <a
+                className="text-hyperlink"
+                href="https://app.powerbi.com/groups/me/reports/b3cec343-cd33-477d-a68e-e1e46acac500/ReportSectionc95f11ff06a200a919a4?experience=power-bi"
+                target="_blank"
+                rel="noreferrer noopener"
+              >
+                CRYSTAL
```
Comment (ellisrichardj) [inline app/front_end/src/components/HelpSupport/HelpSupport.jsx:169]: CRYSTAL was the name of the deprecated system - this should be changed to 'Lab Test Reports'

**Review by sainab-omar — APPROVED** [review]
Looks great, excited to see this live for the users 👍

File: app/viewbovis_data.py [hunk 3]
```diff
@@ -157,7 +157,10 @@ def _transform_dateformat(self, date: str) -> str:
             re.subn(rf'\b(?:{"|".join(month_mapper.keys())})\b',
                     lambda x: month_mapper[x.group()], date)
         try:
-            return datetime.strptime(date_transformed, "%Y-%m-%d").strftime("%d/%m/%Y")
```
Comment (televisedbrain) [inline app/viewbovis_data.py:160]: (No change required) I think this is a redundant change as it would return None either way

**Review by televisedbrain — COMMENTED** [review]
Hi, really good work and I'm excited to see this release coming (: all I'd do is change the gitignore a bit as there's a lot in here I wouldn't expect to see committed. Everything else has been approved. Happy to approve given this.

**Pull request merged**